lib/styles/components/_links.scss

```diff
@@ -1,9 +1,11 @@
+@import '../settings/colors/tokens';
+
 .a-link {
 	cursor: pointer;
-	color: $color-primary;
+	color: $color-primary-text;
```

Comment:
@kvas-damian it was `#0f9698` and now it's `#088289`. I don't think it's correct.

Reply:
there is no other primary color token for text

Reply:
UX is ok with that.